src/Applications/LDAS_App/util/inputs/obs_scaling_params/run_get_L4_Tb_scale_SMAP.m

```diff
@@ -157,7 +161,7 @@
             
             get_model_and_obs_clim_stats( var_name,                              ...
                 run_months, exp_path, exp_run{n}, domain, start_year, end_year, ...
-                dt_assim, species, obs_param, ...
+                dt_assim, t0_assim, species, obs_param, ...
```

Comment:
@gmao-qliu, in this commit, I removed the `_GEOSldas` from the  `get_model_and_obs_clim_stats()`  function.  However, the **4th** call to `get_model_and_obs_clim_stats()` in the run script did **not** have the `_GEOSldas` string in your original code.  Is this a typo?  Or would we need a different version of `get_model_and_obs_clim_stats()` for this case?

Reply:
@gmao-rreichle no need to have a different version. It was not used so I missed editing it with _GEOSldas.